File: Models/slfGameModel.cs
using System.Collections.Generic;

namespace stadtlandfrust.Models
{
    public class SlfGameModel
    {
        public int Id { get; set; }
        public SlfUsersModel GameAdmin { get; set; }

        //holds players and their corresponding scores
        public List<SlfUsersModel> Players {get; set;}

        //Amount of Gamerounds
        public int MaxRounds { get; set; }

        //Current game round
        public int CurrentRound { get; set; }

        //save chars ?

        //categires
        public List<string> Categories { get; set; }

        //gamemode
        public int Gamemode { get; set; }

        public int GameTime { get; set; }

        //indicates wheater stop round after first player is done
        public bool UseTime { get; set; }

    }
}

File: ClientApp/src/components/StadtLandFrustComponents/models/slfGameModel.js
export class SlfGameModel{
    constructor(Id , GameAdmin,Players, MaxRounds,CurrentRound,CategoryValueMap,Gamemode, GameTime,UseTime) {
        this.Id = Id;
        this.GameAdmin = GameAdmin;
        this.Players = Players;
        this.MaxRounds = MaxRounds;
        this.CurrentRound = CurrentRound;
        this.CategoryValueMap = CategoryValueMap;
        this.Gamemode = Gamemode;
        this.GameTime = GameTime;
        this.UseTime = UseTime;
    }
}